api/mesh/v1alpha1/mesh.proto

```diff
@@ -124,11 +124,12 @@ message TracingBackend {
 }
 
 message DatadogTracingBackendConfig {
-  // Address of datadog collector.
+  // Address of datadog collector. Either host:port or unix:/path/to/socket
   string address = 1;
 
-  // Port of datadog collector
-  uint32 port = 2;
+  // If true, when using unix domain socket, path is relative to root of volume
+  // mounted via datadog.tracing.kuma.io/socket-host-path. Default false.
```

Comment:
Can you explain why someone might need this? Is it to do with common datadog deployment patterns?